src/ServiceControl.AcceptanceTests/AcceptanceTest.cs

```diff
@@ -104,11 +104,11 @@ static void RemoveOtherTransportAssemblies(string name)
             }
         }
 
-        protected void ExecuteWhen(Func<bool> execute, Func<IDomainEvents, Task> action, string instanceName = Settings.DEFAULT_SERVICE_NAME)
+        protected Task ExecuteWhen(Func<bool> execute, Func<IDomainEvents, Task> action, string instanceName = Settings.DEFAULT_SERVICE_NAME)
         {
             var timeout = TimeSpan.FromSeconds(1);
 
-            Task.Run(async () =>
+            return Task.Run(async () =>
```

Comment:
What's the reason the task cannot be discarded here? That would be the most straightforward approach to remove the suppression. Furthermore it would allow the method to remain void returning. In addition to that it would require the caller of that high-level Api to know that the task has to be dropped due to its long idling until condition has been met nature